Sources/NIO/ByteBuffer-core.swift

```diff
@@ -550,6 +550,13 @@ public struct ByteBuffer {
             return false
         }
 
+        if self._readerIndex == self.writerIndex {
```

Comment:
Why are we hitting different indices here, one underscored and one not?

Reply:
because I have fat fingers 🤤 ... Let me fix it.

Reply:
fixed